app/jobs/application_job.rb

```diff
@@ -1,4 +1,9 @@
 # frozen_string_literal: true
 
 class ApplicationJob < ActiveJob::Base
+  around_perform do |_, block|
```

Comment:
It's interesting that you needed to add this. Is this documented by Chewy as something you need to do?

Reply:
Ah, this was a workaround for when Chewy was version 0.10.0. Since #1582 merged I can remove this.